ehcache-jcache/src/main/java/org/ehcache/jcache/JCacheConfiguration.java

```diff
@@ -36,6 +34,8 @@
 import javax.cache.integration.CacheLoader;
 import javax.cache.integration.CacheWriter;
 
+import net.sf.ehcache.config.CacheConfiguration;
```

Comment:
sorry about the import change, forgot to flip that IDE automagic off when committing this.